src/SourceGenerators/Uno.UI.SourceGenerators/BindableTypeProviders/DependencyObjectAvailabilityGenerator.cs

```diff
@@ -90,17 +88,15 @@ internal void Generate(GeneratorExecutionContext context)
 
 						modules = modules.Concat(context.Compilation.SourceModule);
 
-						var bindableTypes = from module in modules
-											from type in module.GlobalNamespace.GetNamespaceTypes()
-											where (
-												type.GetAllInterfaces().Any(i => SymbolEqualityComparer.Default.Equals(i, _dependencyObjectSymbol))
-												|| additionalLinkerHintSymbols.Contains(type)
-											)
-											select type;
+						var propertyNames = (from module in modules
+											 from type in module.GlobalNamespace.GetNamespaceTypes()
+											 where (
+												 type.GetAllInterfaces().Any(i => SymbolEqualityComparer.Default.Equals(i, _dependencyObjectSymbol))
+												 || additionalLinkerHintSymbols.Contains(type)
+											 )
+											 select LinkerHintsHelpers.GetPropertyAvailableName(type.GetFullMetadataName())).ToArray();
```

Comment:
The two usages of `bindableTypes` were doing the same thing, producing full metadata name and using that only. So, to avoid the duplicate work, I'm calling it once here.